ChangeLog

```diff
@@ -1,3 +1,428 @@
+* Mon Oct 30 2023 Ken Gaillot <kgaillot@redhat.com> Pacemaker-2.1.7-rc1
+- 1345 commits with 358 files changed, 23622 insertions(+), 17197 deletions(-)
+
+- Features added since Pacemaker-2.1.6
+  + CIB: deprecate "ordering" attribute of "resource_set"
+  + CIB: new cluster option "node-pending-timeout" (defaulting to 2 hours) to
+    fence nodes that do not join Pacemaker's controller group within this much
+    time after joining the cluster
+  + controller: PCMK_node_start_state now works with Pacemaker Remote nodes
+  + crm_verify: add --quiet option (currently same as default behavior)
+  + tools: crm_node supports standard --output-as/--output-to arguments
+  + tests: CTSlab.py was renamed to cts-lab
+
+- Fixes since Pacemaker-2.1.6
+  + logging: restore ability to enable XML trace logs by file and function
+    (regression introduced in 2.1.6)
+  + scheduler: avoid double free with disabled recurring actions
+    (regression introduced in 2.1.5)
+  + tools: consider dampening argument when setting values with attrd_updater
+    (regression introduced in 2.1.5)
+  + tools: wait for reply from crm_node -R (regression introduced in 2.0.5)
+  + attrd: avoid race condition at writer election
```

Comment:
Since we're in the process of reverting this, it probably shouldn't be in the changelog.

Reply:
whoops, thought that was a different thing